### PR TITLE
fix(sign-up): clean phone by adding prefix if not present

### PR DESCRIPTION
**Pull request by Jisay**

### CODE REVIEW

File: packages/manager/apps/sign-up/src/routing.js [hunk 1]
```diff
@@ -55,6 +55,11 @@ export const state = {
     finishSignUp: /* @ngInject */ ($window, getRedirectLocation, me, signUp) => () => signUp
       .saveNic(me.model)
       .then(() => {
+        // for tracking purposes
```
Comment (FredericEspiau) [inline packages/manager/apps/sign-up/src/routing.js]: Tracking what ? And how ?

**Comment by ovh-cds** [comment]
CDS Report sign-up#5327.0 ✘  
* 
  * Build CA ✘  
  * Build EU ✘  
  * Build US ✘

File: packages/manager/apps/sign-up/src/index.html [hunk 2]
```diff
@@ -13,8 +13,9 @@
             <div class="py-3 py-md-5 px-3">
                 <a class="d-inline-block pb-3 pb-md-5"
                    data-ng-href="{{ $ctrl.logoUrl }}">
-                    <img src="assets/img/logo-ovh.svg"
-                         alt="OVHcloud - Innovation for freedom" />
+                    <img data-ng-src="{{ $ctrl.logoSrc }}"
```
Comment (FredericEspiau) [inline packages/manager/apps/sign-up/src/index.html]:
```suggestion
                    <img data-ng-src="{{:: $ctrl.logoSrc }}"
```

File: packages/manager/apps/sign-up/src/constants.js [hunk 3]
```diff
@@ -23,11 +23,14 @@ export const HOME_PAGE = {
   WS: 'https://www.ovh.com/us/es/',
 };
 
+export const OVH_LOGO = 'https://eu.api.ovh.com/images/com-square-bichro.png';
```
Comment (FredericEspiau) [inline packages/manager/apps/sign-up/src/constants.js]: Shouldn't we base64 it instead ?

**Pull request merged**